components/xip-patch/xip-patch.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+EXTERNAL_PUBLIC_IP=$(kubectl get service -n istio-system istio-ingressgateway -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
```

Comment:
Are we sure this will be available? Provisioning of LoadBalancers take some time.